platforms/cuda/src/CudaKernels.cpp

```diff
@@ -2045,8 +2185,80 @@ double CudaCalcNonbondedForceKernel::execute(ContextImpl& context, bool includeF
             cuEventRecord(pmeSyncEvent, pmeStream);
             cu.restoreDefaultStream();
         }
+
+        // As written, we check only the Electrostatic grid pointer to get here.  We could separate them out, but for
+        // now we assume that LJPME can only be used if electrostatic PME is also active.
+        if(doLJPME){
+            if (usePmeStream)
+                cu.setCurrentStream(dispersionPmeStream);
```

Comment:
Thanks for the feedback.  Yeah, I went back and forth about whether to make a new stream or not, and eventually decided that having the extra stream to overlap communication and computation would be worth trying.  I'll consolidate the memory and streams, as you suggested.